seqr/models.py

```diff
@@ -36,6 +36,16 @@ def _slugify(text):
     return __slugify(text).replace('-', '_')
 
 
+class AnvilUser(models.Model):
+    user = models.OneToOneField(User, on_delete=models.CASCADE)
+    anvil_username = models.CharField(db_index=True, max_length=100, null=True, blank=True) # an email address
```

Comment:
I revisited the AnVIL and Terra API document. You are right. They always use `email` rather than `username`. To keep consistency with AnVIL, I will just use `email` without `anvil` and `user` since the model name is `AnvilUser` already.